<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,2">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="DelButton">
       <property name="text">
        <string>Delete File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AddButton">
       <property name="text">
        <string>Add FIle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BackButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>Path:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit"/>
     <widget class="QPushButton" name="FindPathButton">
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QTreeWidget" name="LeftTree">
     <property name="lineWidth">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTreeView" name="PanelTree"/>
   </item>
   <item row="2" column="1">
    <widget class="QTreeWidget" name="RightTree">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
